core/flyout_extension_category_header.js

```diff
@@ -81,7 +81,8 @@ Blockly.FlyoutExtensionCategoryHeader.prototype.createDom = function() {
   var marginX = 15;
   var marginY = 10;
 
-  var statusButtonX = (this.flyoutWidth_ - statusButtonWidth - marginX) / this.workspace_.scale;
+  var statusButtonX = this.workspace_.RTL ? (marginX - this.flyoutWidth_ + statusButtonWidth) :
+      (this.flyoutWidth_ - statusButtonWidth - marginX) / this.workspace_.scale;
```

Comment:
Why are you dividing by scale in the LTR case but not the RTL case?

Reply:
Yeah, that's what @chrisgarrity and I were trying to figure out too. For some reason, not dividing by the scale in the LTR case displays the extension status button part of the way through the width of the flyout instead of at the right edge of the flyout. 

In the RTL case, dividing by the workspace scale does the analogous thing.

Our guess was that somewhere before this point, the flyout width is already being divided by the workspace scale but only in the RTL case... but we couldn't find it.

Reply:
I think it's related to how we calculate the moveTo x in flyout_vertical below. It's using flyoutWidth. That flyoutWidth has already had the scale factor applied: https://github.com/LLK/scratch-blocks/blob/c21757220dbda20098fd7ee25a3232671bcb316d/core/flyout_vertical.js#L448

Reply:
Ah, and that value of flyoutWidth is only being used for RTL.

But then it looks like most (all?) parts of statusButtonX are being negated in that moveTo below, so I'm still very confused about what this code is doing.
Is this value of statusButtonX the

Reply:
Basically, I think the statusButtonX calculation is just placing the status button to the left of the `Extensions` label (instead of to the right, like it is in LTR), and then the moveTo is moving the *entire* label (actual status button + the word `Extensions`) into the right place on the flyout